ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -289,25 +342,38 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         super.layoutSubviews()
 
         accessoryView?.sizeToFit()
+        leadingView?.sizeToFit()
 
-        let titleWidth = contentView.frame.width - (Constants.horizontalMargin + TableViewHeaderFooterView.titleRightOffset(accessoryView: accessoryView))
+        let titleWidth = contentView.frame.width - (TableViewHeaderFooterView.titleRightOffset(accessoryView: accessoryView) + TableViewHeaderFooterView.titleLeftOffset(leadingView: leadingView))
+        var titleXOffset = Constants.horizontalMargin
         let titleHeight: CGFloat
         let titleYOffset: CGFloat
         switch style {
-        case .header, .footer:
-            titleYOffset = style == .header ? Constants.titleDefaultTopMargin : Constants.titleDefaultBottomMargin
+        case .header, .footer, .headerPrimary:
+            titleYOffset = style == .footer ? Constants.titleDefaultBottomMargin : Constants.titleDefaultTopMargin
             titleHeight = contentView.frame.height - Constants.titleDefaultTopMargin - Constants.titleDefaultBottomMargin
         case .divider, .dividerHighlighted:
             titleYOffset = Constants.titleDividerVerticalMargin
             titleHeight = contentView.frame.height - (Constants.titleDividerVerticalMargin * 2)
         }
+        
+        if let leadingView = leadingView {
+            let xOffset = Constants.accessoryViewMarginLeft
+            let yOffset = contentView.frame.height - leadingView.frame.height - Constants.titleDefaultBottomMargin
+            leadingView.frame = CGRect(
+                origin: CGPoint(x: xOffset, y: yOffset),
+                size: leadingView.frame.size
+            )
+            titleXOffset += leadingView.frame.width
+        }
+        
         titleView.frame = CGRect(
-            x: Constants.horizontalMargin,
+            x: titleXOffset,
```

Comment:
might have to be different offset for RTL ?

Reply:
Checked for RTL, it is not affected